roles/os_zabbix/library/openshift_tools

```diff
@@ -0,0 +1 @@
+../../../../openshift-tools/openshift_tools
```

Comment:
This should just be for testing. For actual execution, we want to use the one installed by the rpm.